plugin/src/main/java/io/jenkins/plugins/coverage/model/util/FractionFormatter.java

```diff
@@ -30,6 +30,20 @@ public static Fraction transformFractionToPercentage(final Fraction fraction) {
         return fraction.multiplyBy(HUNDRED);
     }
 
+    /**
```

Comment:
Currently it is somewhat unclear throughout the whole plugin whether a `Fraction` instance ist used as a rational number or a percentage. Maybe it would make sense to create a new `Percentage` class that has a `Fraction` (multiplied by 100) as a field. Does that make more sense?

Reply:
I added a class `CoveragePercentage` which can be used for displaying and serializing coverage values as percentage.
`Fraction` is still in use for calculations since it can represent coveraged and missed lines properly and also provides math operations.
Now, `Fraction` is not used for displaying or serializing coverage anymore and it is clear whether a fraction or a percentage is represented by a value.